### DISCUSSION TITLE
Huion Q11k not being detected

### DISCUSSION THREAD

**Discussion by Novaer213** [OpenTabletDriver/OpenTabletDriver#1308] [Q&A] [answered]
As the title suggests, my huion q11k is not being detected by OTD. Please, is there a fix to this?

edit: It's been 10 days, no one has found a fix yet?

**Comment by jamesbt365** [comment]
Try https://github.com/X9VoiD/WinUSBCleanup/releases/tag/v1.0

**Comment by gonX** [comment, marked as answer]
Please see https://opentabletdriver.net/Wiki/FAQ/Windows#troubleshooting if it is still not working for you.
